tests/python/unittest/test_sparse_operator.py

```diff
@@ -1675,6 +1675,30 @@ def check_sparse_embedding(in_dim, out_dim, batch, densities, deterministic):
     check_sparse_embedding(in_dim, out_dim, batch, densities, False)
 
 
+@with_seed()
+def test_sparse_broadcast_mul_div():
+    from scipy.sparse import random, csr_matrix
+    def check_broadcast_mul(mx_lhs, mx_rhs, np_lhs, np_rhs, dtype):
+        assert_almost_equal(mx.nd.broadcast_mul(mx_lhs, mx_rhs).asnumpy(), np.multiply(np_lhs, np_rhs), atol=1e-4)
+    def check_broadcast_div(mx_lhs, mx_rhs, np_lhs, np_rhs, dtype):
+        assert_almost_equal(mx.nd.broadcast_div(mx_lhs, mx_rhs).asnumpy(), np.divide(np_lhs, np_rhs), atol=1e-4)
+    shape = (4,3)
+    np_lhs = random(shape[0], shape[1], density=0.25, dtype=np.float32).tocsr()
+    mx_lhs = mx.nd.sparse.csr_matrix((np_lhs.data, np_lhs.indices, np_lhs.indptr), shape=shape)
```

Comment:
Why not just use `test_utils.rand_ndarray(shape, stype, density)`?

Reply:
Done, was not aware of that helper function